--- pkg/apis/app/v1alpha1/podset_types.go	2019-08-04 07:04:29.291741151 +0000
+++ pkg/apis/app/v1alpha1/podset_types.go.v1	2019-08-04 08:02:48.799483746 +0000
@@ -10,17 +10,13 @@
 // PodSetSpec defines the desired state of PodSet
 // +k8s:openapi-gen=true
 type PodSetSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	Replicas int32 `json:"replicas"`
 }
 
 // PodSetStatus defines the observed state of PodSet
 // +k8s:openapi-gen=true
 type PodSetStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	PodNames []string `json:"podNames"`
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
